sample-compose/app/src/main/java/co/nimblehq/sample/compose/navigation/Navigation.kt

```diff
@@ -0,0 +1,38 @@
+package co.nimblehq.sample.compose.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavType
+import androidx.navigation.compose.*
+import androidx.navigation.navArgument
+import co.nimblehq.sample.compose.navigation.NavigationRoute.HOME_ROUTE
+import co.nimblehq.sample.compose.navigation.NavigationRoute.SECOND_ARGUMENT_ID
+import co.nimblehq.sample.compose.navigation.NavigationRoute.SECOND_ROUTE
+import co.nimblehq.sample.compose.ui.screens.home.HomeScreen
+import co.nimblehq.sample.compose.ui.screens.second.SecondScreen
+
+object NavigationRoute {
+    const val HOME_ROUTE = "home"
+    const val SECOND_ROUTE = "second"
+
+    const val SECOND_ARGUMENT_ID = "id"
+}
+
+@Composable
+fun Navigation() {
+    val navController = rememberNavController()
+    NavHost(navController = navController, startDestination = HOME_ROUTE) {
+        composable(route = HOME_ROUTE) {
+            HomeScreen(navController = navController)
+        }
+        composable(
+            route = SECOND_ROUTE + "/{${SECOND_ARGUMENT_ID}}",
```

Comment:
Why do we need to add `{}` for this variable?
```suggestion
            route = SECOND_ROUTE + "/{$SECOND_ARGUMENT_ID}",
```

Reply:
Resolved: https://github.com/nimblehq/android-templates/discussions/279

Reply:
@chornerman I'm not sure what you mean with the `Resolved: #279` here? Did you mean to resolve this once open the PR for #279 issue, or have you resolved it already in this PR?

Reply:
@lydiasama I mean this comment is outdated because I updated the PR according to the #279 discussion 😄

Reply:
I see. So I think it sounds to be more clear if we say: `Resolve in #279`. 🤔 The `Resolved` meant we have resolved this issue already to me. 🙏🏻